src/share/doc/apidiff.md

```diff
@@ -497,3 +497,18 @@ The tool does _not_ attempt to run `javadoc` locally to generate the page on the
 [jdk.compiler]: https://docs.oracle.com/en/java/javase/17/docs/api/jdk.compiler/module-summary.html
 [Java Language Model API]: https://docs.oracle.com/en/java/javase/17/docs/api/java.compiler/javax/lang/model/package-summary.html
 [Compiler Tree API]: https://docs.oracle.com/en/java/javase/17/docs/api/jdk.compiler/com/sun/source/doctree/package-summary.html
+
+## Example
+
+To compare APIs in the `java.base` module in JDK builds `/local/baseline-jdk` and `/local/updated-jdk`, and place the result in the directory `out`, run the following command:
+
+```
+bin/apidiff --api jdk \
+    --jdk-build=/local/baseline-jdk/build/linux-x86_64-server-fastdebug --jdk-docs docs \
+    --api sv \
+    --jdk-build=/local/updated-jdk/build/linux-x86_64-server-fastdebug --jdk-docs docs \
+    -d out \
+    --compare-api-descriptions true \
```

Comment:
It is intended that `--compare-api-descriptions` should default to `true`, although it is not wrong to be explicit.  If you found it to be necessary, that would be worth investigating.

Reply:
> It is intended that `--compare-api-descriptions` should default to `true`, although it is not wrong to be explicit. If you found it to be necessary, that would be worth investigating.

I find it very hard to understand the ifs and buts in the description of this option, and @alanbateman used the option this way. Please advise me to leave it in or take it out.

Reply:
I used it in the example I gave Andrew to be explicit, no other reason. In any case, I agree we need some recipes in the docs. If they include a path to a build then probably better to use a path to a release build as someone is unlikely to be using fast debug build for this kinda of thing.

Reply:
@AlanBateman @theRealAph 
I will look to create more examples.

Reply:
> @AlanBateman @theRealAph I will look to create more examples. 

Will you approve this one, please?